src/DynamoSandbox/DynamoCoreSetup.cs

```diff
@@ -102,10 +102,19 @@ public void RunApplication(Application app)
 
                     if (viewModel != null)
                     {
-                        // Show the unhandled exception dialog so user can copy the 
-                        // crash details and report the crash if she chooses to.
-                        viewModel.Model.OnRequestsCrashPrompt(null,
-                            new CrashPromptArgs(e));
+
+
+                        if (CrashReportTool.IsCEREnabled())
+                        {
+                            CrashReportTool.OnCrashReportWindow(new CrashReportArgs(viewModel));
+                        }
+                        else
+                        {
+                            // Show the unhandled exception dialog so user can copy the 
+                            // crash details and report the crash if she chooses to.
+                            viewModel.Model.OnRequestsCrashPrompt(null,
+                                new CrashPromptArgs(e));
```

Comment:
So now it's going to be the CER reporting dialog or our crash dialog in Dynamo - one or the other? Can it be both? In other words will the information we see in the standard Dynamo crash dialog be useful to show when the CER tool is enabled as well?

Reply:
I would say one or the other. 
CER collects everything that the old crash dialog was collecting, and more...